tools/template_based_query_generation/src/main/java/Utils.java

```diff
@@ -50,10 +51,10 @@ public static String getRandomString(int length) throws IllegalArgumentException
 
 		for (int i = 0; i < length; i++) {
 			int randomIndex = (int) (random.nextDouble() * CHARSET.length());
-			char randomChar = CHARSET.charAt(randomIndex);
+			char randomChar = CHARSET.charAt(random.nextInt(0, 63));
```

Comment:
Sorry for asking for change back and forth. I guess it would be better to use CHARSET.charAt(random.nextInt(0, CHARSET.length())) here.

Reply:
Victor made this change.

Reply:
Fixed.